src/modules/carts/entities/cart.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Product_list } from 'src/modules/product_lists/entities/product_list.entity';
+import { User } from 'src/modules/users/entities/user.entity';
```

Comment:
```suggestion
import { User } from '../../../users/entities/user.entity';
```

не використовуй `src`